app/assets/javascripts/alerts.js

```diff
@@ -3,6 +3,9 @@ $(window).load(function() {
     $(function () {
         var alert = $('.alert');
         if (alert.length > 0) {
+            if (alert.hasClass('permanent')) {
```

Comment:
I would encourage you to change this to `if (alert.length > 0 && !alert.hasClass('permanent') {`. This is my personal opinion, but using return statements for control flow can make things hard to understand (plus it'd be 3 lines shorter!)